bridge/core_rs/src/dom.rs

```diff
@@ -11,7 +11,7 @@ pub fn init_webf_dom(context: &ExecutingContext){
   let exception_state = context.create_exception_state();
 
   let html_tag_name = CString::new("html");
-  let html_element = document.create_element(&html_tag_name.unwrap(), &exception_state)?;
+  let html_element = document.create_element(&html_tag_name.unwrap(), &exception_state).unwrap();
```

Comment:
Is your Rust version too old?

The operator is a shortcut for the unwrap() function